core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -385,5 +385,9 @@ oppia.controller('SettingsTab', [
     $scope.closeRolesForm = function() {
       $scope.isRolesFormOpen = false;
     };
+
+    $scope.isTitlePresent = function() {
+      return ExplorationTitleService.displayed.length > 0;
```

Comment:
I think we should look for the savedValue instead of displayed value. (Let me know if it's not clear)

Reply:
Im not sure If i understood this correctly, Can you please explain. (I tried searching for savedValue but could not find anything related to it.

Reply:
@amey-kudari, I was talking about the savedMemento value, as displayed values are not always the correct values.

Reply:
Alright, I agree I'll check what the savedMomento is